server/storage/types.go

```diff
@@ -4,7 +4,29 @@ import "github.com/docker/notary/tuf/data"
 
 // MetaUpdate packages up the fields required to update a TUF record
 type MetaUpdate struct {
-	Role    data.RoleName
-	Version int
-	Data    []byte
+	Role     data.RoleName
+	Version  int
+	Data     []byte
+	Channels []*Channel
```

Comment:
It looks like, from the memory implementation (I am not very familiar with advanced SQL syntax like joins and grouping, so I hadn't gotten to looking at the SQL implementation yet :)) that if multiple channels are passed, the same data will be added to all the channels.

Out of curiosity, what is the use case for staging the same data to multiple channels at once?  In the case of published and staging, you probably don't need to write to both.  In the case where each channel is a particular user or key ID, it also seems strange to write to multiple key IDs.  Is the case for writing to both staging and a particular key ID?  (I was a little fuzzy on how that would work for threshold signing).

If that's the case, and adding the same update should apply to every channel in the channel list (as opposed to the `Get*` functions, for which if multiple channels are passed only the data from the first channel that gun and role are in will be returned), should this be documented in the storage interface's `UpdateCurrent` function declaration?  And in `server/storage/storage_test.go`'s `testUpdateCurrentInChannel`, can we update multiple channels at the same time and ensure that getting for both channels returns the same data?